common/formatting/align.h

```diff
@@ -66,32 +80,56 @@ struct ColumnPositionEntry {
 // and call ReserveNewColumn() in locations that want a new columns.
 class ColumnSchemaScanner : public TreeContextPathVisitor {
  public:
-  ColumnSchemaScanner() = default;
+  ColumnSchemaScanner()
+      : sparse_columns_(ColumnPositionTree({{}, TokenInfo::EOFToken(), {}})) {}
 
   // Returns the collection of column position entries.
-  const std::vector<ColumnPositionEntry>& SparseColumns() const {
-    return sparse_columns_;
-  }
+  const ColumnPositionTree& SparseColumns() const { return sparse_columns_; }
 
  protected:
+  // Returns subpath relative to base_path
+  static SyntaxTreePath GetSubpath(
+      const SyntaxTreePath& base_path,
+      std::initializer_list<SyntaxTreePath::value_type> subpositions) {
+    auto subpath = base_path;
+    subpath.insert(subpath.end(), subpositions);
+    return subpath;
+  }
+
   // Mark the start of a new column for alignment.
+  // 'parent_column' is a reference to the parent column.
```

Comment:
Also, we need to now document the return value new in this change.

Reply:
Fixed in [parent PR](https://github.com/chipsalliance/verible/pull/865)